css/style.css

```diff
@@ -1,4 +1,144 @@
-h1 {
-  font-family: Arial, Helvetica, sans-serif;
-  color: rgb(50, 0, 200);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main {
+  background: url(../img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  box-sizing: border-box;
+  border-radius: 8px;
+}
+
+.button:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.button:active {
+  background: #2230d2;
+}
+
+.button:disabled {
+  border: 1px solid #c1c7d0;
+}
+
+.main h2 {
+  flex: none;
+  margin: 12px 0;
+}
+
+.mobile-menu {
+  display: none;
+  content: url(../img/icon-menu.png);
+}
+
+.intro {
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172840;
+}
+
+.description {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  color: #344563;
+}
+
+.social {
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 500;
+  color: #7f8cff;
+}
+
+.social-icons {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 20px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-item {
+  margin: 0 10px;
+}
+
+.nav-link {
+  font-size: 15px;
+  font-weight: 500;
+  color: #344563;
+}
+
+.nav-link:hover {
+  color: #482ff7;
+}
+
+.nav-logo {
+  font-size: 18px;
+  font-weight: bold;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+@media only screen and (max-width: 768px) {
+  .nav-menu {
+    display: none;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
+
+  .nav-item {
+    margin: 40px 0;
+  }
+
+  .mobile-menu {
+    display: block;
+    cursor: pointer;
+  }
 }
```

Comment:
- [ ] **REQUIRED:** This application is building according to the `Mobile First` Approach, so there is no need for media queries and invisible parts